### PR TITLE
CONTRIBUTING.md: No need to mention java and scala directory separation 

### LINKED ISSUE #24056: CONTRIBUTING.md doesn't need to mention scala java directory separation
I will submit a pull request.

### PR DESCRIPTION
**Pull request by richardimaoka**
Closes #24056

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ The steps are exactly the same for everyone involved in the project (be it core
...
 1. Once your feature is complete, prepare the commit following our [Creating Commits And Writing Commit Messages](#creating-commits-and-writing-commit-messages). For example, a good commit message would be: `Adding compression support for Manifests #22222` (note the reference to the ticket it aimed to resolve).
-1. If it's a new feature, or a change of behaviour, document it on the [akka-docs](https://github.com/akka/akka/tree/master/akka-docs), remember, an undocumented feature is not a feature. If the feature was touching Scala or Java DSL, make sure to document it in both the Java and Scala documentation (usually in a file of the same name, but under `/scala/` instead of `/java/` etc).
+1. If it's a new feature, or a change of behaviour, document it on the [akka-docs](https://github.com/akka/akka/tree/master/akka-docs), remember, an undocumented feature is not a feature. If the feature was touching Scala or Java DSL, make sure to document it in both the Java and Scala documentation.
```
Comment (johanandren) [inline CONTRIBUTING.md]: Instead of "If the feature .... .Java and Scala documentation." maybe "Make sure to document both the Scala and Java APIs."

**Review by johanandren — COMMENTED** [review]
Looks good but added a minor thing.

File: CONTRIBUTING.md (hunk 1 above)
Comment (richardimaoka) [reply CONTRIBUTING.md]: Sure, updated.

Comment (johanandren) [reply CONTRIBUTING.md]: Thanks

**Review by johanandren — APPROVED** [review]

**Comment by raboof** [comment]
Nicely cleans up! :)

**Pull request merged**